Detections/DSMAzureBlobStorageLogs/AccessToSensitiveDataByTerminatedEmployees.yaml

```diff
@@ -0,0 +1,88 @@
+id: 5feef89a-d632-4294-a05c-988a1c64da23
+name: Access to sensitive data by Terminated Employees
+description: |
+  This query detects access to data that has classification insights (link below) and/or sensitivity label insights (link below), by users that are marked as terminated employees.
+  Classification insights: https://docs.microsoft.com/en-us/azure/purview/classification-insights
+  Sensitivty label insigths: https://docs.microsoft.com/en-us/azure/purview/sensitivity-insights
+requiredDataConnectors:
+  - connectorId: ThreatIntelligence
+    dataTypes:
+      - ThreatIntelligenceIndicator
+  - connectorId: ThreatIntelligenceTaxii
+    dataTypes:
+      - ThreatIntelligenceIndicator
+severity: High
+queryPeriod: 1d
+queryFrequency: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Exfiltration
+  - Collection
+relevantTechniques:
+  - T1567
+  - T1530
+query: |
+
+  // accesses to labeled and/or classified data
+  let AccessLogs = DSMAzureBlobStorageLogs
+      | where isnotempty(CorrelationId)
+      | project
+          Asset = tostring(parse_url(Uri).Path),
+          UserObjectId=RequesterObjectId,
```

Comment:
RequesterObjectId seems to be present in the custom table that you have defined above but does not seem to be present in the schema that I checked in one of the LA  instance that I have access to. Can you please double check to ensure that this is accurate field.
![image](https://user-images.githubusercontent.com/37783395/160109597-09bf0e06-c56a-466f-b87b-f862f0d2df7b.png)

Reply:
This difference is actually weird. Could you tell where is the table you were querying?
Thanks

<img width="433" alt="image" src="https://user-images.githubusercontent.com/38659203/160416611-f5d14e41-249e-44c7-b754-c1c0f7fd5dcc.png">

Reply:
@amramz - As discussed, we will want to do the validation bypass for now.  There is already one example for the same reason you should put here - https://github.com/Azure/Azure-Sentinel/blob/master/.script/tests/KqlvalidationsTests/SkipValidationsTemplates.json - you would update the same file, but with the ID from the top of your YAML file.  Basically, add this to the end of the SkipValidationsTemplates.json file I list above -
  {
    "id": "5feef89a-d632-4294-a05c-988a1c64da23",
    "templateName": "AccessToSensitiveDataByTerminatedEmployees.yaml",
    "validationFailReason": "The name '_GetWatchList' does not refer to any known function"
  }